diff -ruN monotheka-0.0.5./Plugins/AVIGet/AVIGet.cs monotheka-0.0.5/Plugins/AVIGet/AVIGet.cs
--- monotheka-0.0.5./Plugins/AVIGet/AVIGet.cs	2005-04-21 01:43:59.000000000 +0200
+++ monotheka-0.0.5/Plugins/AVIGet/AVIGet.cs	2005-08-04 23:26:49.100507720 +0200
@@ -68,7 +68,19 @@
                 // RIFF stuff
                 RIFFReader __RR = new RIFFReader(__AVILocation.GetResult());
                 __RR.Run();
-                return __RR.GetMovie();
+
+                // If movie title is empty, set it to filename
+                Movie __M = __RR.GetMovie();
+                if (_M.Data[(int) MovieField.TITLE].Equals(""))
+                {
+                        string __title = Path.GetFileNameWithoutExtension(__AVILocation.GetResult());
+                        __title = __title.Replace("_", " ");
+                        __title = __title.Replace(".", " ");
+
+                        __M.Data[(int) MovieField.TITLE] = __title;
+                }
+
+                return __M;
             }
         } catch (RIFFReaderException __Excp) {
             new Util.Windows.OKDialogWindow(__Excp.Message , null, MessageType.Warning);
