diff -ruN monotheka-0.0.5/Util/Net.cs monotheka-0.0.5./Util/Net.cs
--- monotheka-0.0.5/Util/Net.cs	2005-04-10 14:12:15.000000000 +0200
+++ monotheka-0.0.5./Util/Net.cs	2005-08-04 20:12:05.000000000 +0200
@@ -187,7 +187,7 @@
                 __Command = Strings.ClearString(__Command);
                 
                 // parse url
-                _Url = _Url.Replace(" ", "%20");
+                _Url = "\"" + _Url.Replace(" ", "%20") + "\"";
                                 
                 try {
                     System.Diagnostics.Process proc = new System.Diagnostics.Process(); 
